tests/test_benchmark/test_benchmark.py

```diff
@@ -51,6 +52,8 @@ def test_prompt_name_passed_to_all_encodes(task_name: str | mteb.AbsTask):
     _task_name = (
         task_name.metadata.name if isinstance(task_name, mteb.AbsTask) else task_name
     )
+    if isinstance(task_name, AbsTaskInstructionRetrieval):
```

Comment:
Are we skipping the test due to time?

Reply:
It seems like it is since they overwrite the prompt_name kwargs? I assume that is not what we want (given above comment)